data.sql

```diff
@@ -30,3 +31,74 @@ VALUES ( 10, 'Blossom', '1998-10-13', 3, 'true', '17', '' );
 VALUES ( 11, 'Ditto', '2022-05-14', 4, 'true', '22', '' );
+
+UPDATE animals SET Species = 'Unspecified'; ----> sets all the species column valuesf to unspecified
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Squirtle'; 
+
+UPDATE animals SET Date_of_birth = '2005-06-12' WHERE Name = 'Angemon';
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Pikachu'; -----> this is a request
+
+UPDATE animals SET Species = 'digimon' WHERE Name like '%mon'; ----> Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+
+UPDATE animals SET Species = 'pokemon' WHERE Species = 'Unspecified'; ----> Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals SET Weight_kg = Weight_kg * -1;
+
+UPDATE animals SET Weight_kg = Weight_kg * -1 WHERE Weight_kg < 0 ;
+
+UPDATE animals SET species_id = 'Unspecified';
+
+UPDATE animals SET species_id = 'digimon' WHERE Name like '%mon';
+
+UPDATE animals SET species_id = 'pokemon' WHERE species_id = 'Unspecified';
+
+UPDATE animals SET owner_id = 'Sam Smith' WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 'Sam Smith' WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 'Jennifer Orwell' WHERE name = 'Gabumon';
+UPDATE animals SET owner_id = 'Jennifer Orwell' WHERE name = 'Pikachu';
+UPDATE animals SET owner_id = 'Bob' WHERE name = 'Devimon';
+UPDATE animals SET owner_id = 'Bob' WHERE name = 'Plantmon';
+UPDATE animals SET owner_id = 'Melody Pond' WHERE name = 'Charmander';
+UPDATE animals SET owner_id = 'Melody Pond' WHERE name = 'Squirtle';
+UPDATE animals SET owner_id = 'Melody Pond' WHERE name = 'Blossom';
+UPDATE animals SET owner_id = 'Dean Winchester' WHERE name = 'Angemon';
+UPDATE animals SET owner_id = 'Dean Winchester' WHERE name = 'Boarmon';
+
+
+
+-----OWNERS TABLE-----
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 1, 'Sam Smith', 34 );
+
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 2, 'Jennifer Orwell', 19 );
+
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 3, 'Bob', 45 );
+
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 4, 'Melody Pond', 77 );
+
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 5, 'Dean Winchester', 14 );
+
+INSERT INTO owner ( ID, Full_name, Age )
+VALUES ( 6, 'Jodie Whittaker', 38 );
+
+-----SPECIES-----
+INSERT INTO species ( ID, Name )
+VALUES ( 1, 'Pokemon' );
+
+INSERT INTO species ( ID, Name )
+VALUES ( 2, 'Digimon' );
+
+
+----INNER JOIN----
+
+SELECT owner.full_name, animals.name FROM animals INNER JOIN owner ON owner.full_name = animals.owner_id WHERE owner.full_name='Melody Pond';
+SELECT species.name, animals.name FROM animals INNER JOIN species ON species.name = animals.species_id WHERE species.name='Pokemon';
+SELECT owner.full_name, animals.name FROM animals INNER JOIN owner ON owner.full_name = animals.owner_id;
+SELECT COUNT(*) FROM animals WHERE species_id = 'Pokemon';
```

Comment:
- [ ] If it is done correctly, `species_id` has a data type of `INT` though it is assigned a `String` value of `Pokeman`. What you need to do achieve this selection is to make a left join between the `animals`  and species table based on the **animals.species_id = species.id** and then check if **species.name = 'Pokemon'**.